src/seedu/addressbook/AddressBook.java

```diff
@@ -1150,8 +1157,8 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+        return fullString.replace(prefix, "");
```

Comment:
You need to get the intended purpose of the method correctly.  We need to deal with the case of replacing only the prefix, and a prefix can occur only at the start of fullString.